### PR TITLE
add exclusion for cookie term in experiments.js

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by dvoytenko** [comment]
I this is ok. But at some point, what do you think about?
```
var forbiddenTerms = {
...
'cookie\\W': {
  message: requiresReviewPrivacy,
  whitelist: ['src/experiments.js']
},
...
}
```

**Comment by cramforce** [comment]
@erwinmombay Do you want to make the change @dvoytenko is suggesting?

**Comment by erwinmombay** [comment]
@dvoytenko @cramforce  yep, i was going to do that but the only thing i was worried about was the duplication of the regex term and somebody changing one would have to remember to change the other. with es6 of course we could do { [key]: value } but we dont use es6 in the build tools.

i'll make the change if you guys arent too concerned about the duplication part i mentioned.

**Comment by erwinmombay** [comment]
@dvoytenko @cramforce oops totally misread the recommendation. I'll make the change.

**Comment by erwinmombay** [comment]
@dvoytenko done. PTAL

File: build-system/tasks/presubmit-checks.js [hunk 1]
```diff
@@ -36,7 +36,13 @@ var forbiddenTerms = {
   'it\\.only': '',
   'console\\.\\w+\\(': 'If you run against this, use console/*OK*/.log to ' +
       'whitelist a legit case.',
-  'cookie\\W': requiresReviewPrivacy,
+  'cookie\\W': {
+    message: requiresReviewPrivacy,
+    whitelist: [
+      'src/experiments.js',
```
Comment (dvoytenko) [inline build-system/tasks/presubmit-checks.js]: So, I'm changing implementation a bit. So, could you please submit this with empty `whitelist: []`?

**Comment by dvoytenko** [comment]
LGTM with one comment.

File: build-system/tasks/presubmit-checks.js (hunk 1 above)
Comment (erwinmombay) [inline build-system/tasks/presubmit-checks.js]: will do.

**Pull request merged**